pkg/logs/parser/parser.go

```diff
@@ -0,0 +1,37 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package parser
+
+// ParsedLine represents a containerd message
+type ParsedLine struct {
```

Comment:
ParsedLine is *really* similar to Message, they're almost the same. Can we keep a single one ?